README.md

```diff
@@ -1,68 +1,211 @@
+> Hashes are a lot like arrays: they have a bunch of slots that can point to various objects. However, in an array, the slots are lined up in a row, and each one is numbered (starting from zero). In a hash, the slots aren’t in a row (they are just sort of jumbled together) and you can use any object to refer to a slot, not just a number. 
+> 
+> –– [Chris Pine's Learn to Program, page 108](http://books.flatironschool.com/books/43?page=108)
+
+
+Up until this point, we've stored our data in list-form using arrays. Let's take a look at an example where an array would not be sufficient to meet our storage needs.
+
+### Example: Customs Inspector
+
+![Herman Melville](https://upload.wikimedia.org/wikipedia/commons/thumb/f/f1/Herman_Melville.jpg/160px-Herman_Melville.jpg)
+
+The year is 1866 and we work beside [Herman Melville](https://en.wikipedia.org/wiki/Herman_Melville) (author of Moby Dick) as a customs inspector in New York. (I know, when did our lives become so old-timey and exciting? FYI he really did work there in 1866.) We're in charge of inspecting an incoming shipment and we need to track the items we're inspecting by listing each type of item (for example whale bone corset, porcelain, oil painting, other old fashioned item, etc) and the number of that item in the shipment. 
+
+Let's say our shipment containts 5 whale bone corsets, 2 porcelain vases and 3 oil paintings. If we tried to store them in an array in list form, it would look like this: 
+
+```ruby
+old_fashioned_things = ["whale bone corset", "porcelain vase", "oil painting"
+```
+As it stands, there is no way for us to include the count of each item in a way that associates the item to it's count. That's no good. How is our co-worker Herman Melville going to be able to tell how many of each item are in this shipment? 
+
+Here's where a hash can come in handy. With a hash, we can store a list of associated pairs. In other words, we can store the pairs of item/count. Let's take a look: 
+
+```ruby
+old_fashioned_things = {
+  "whale bone corset" => 5, 
+  "porcelain vase" => 2, 
+  "oil painting" => 3
+}
+```
 
 ## Structure
-* Hashes are structured with keys and values.
+Hashes are structured with keys and values. Each key/value pair makes up one unit in the hash. The entire collection of key/value pairs, which are comma separated, is enclosed in curly braces, `{ }`. 
 
 ```ruby
 hash = {:key => "value", :another_key => "another value"}
 ```
 
-## Creation
-* Making a new hash can be accomplished in a couple different ways.
+You can think of a hash as a dictionary. The key is the reference point that is set equal to an associated value. In fact, a dictionary makes a pretty good hash example. Let's take a look: 
```

Comment:
Also maybe a discussion of what can be used as a key? Answer is anything, but remember keys are unique!